python/pyproject.toml

```diff
@@ -21,7 +21,8 @@ build-backend = "maturin"
 requires = ["maturin>=1.0, <2.0"]
 
 [tool.maturin]
-python-source = "./"
+python-source = "egobox"
```

Comment:
this will make pyo3-stub-gen to put `egobox.pyi` inside `python/egobox`, I hope it is corrected?